src/compiler/interpreter.jl

```diff
@@ -70,11 +70,13 @@ end
 function is_primitive_func(@nospecialize(TT))
     isa(TT, DataType) || return false
     ft = TT.parameters[1]
+    if ft === typeof(Base.string) 
```

Comment:
This was intentionally only for `string` with FloatingPoint args

Reply:
I mean regardless this is inactive and we want to mark as such?

Reply:
Yes but we don't want to change inlining behavior more than necessary